opentech/apply/users/models.py

```diff
@@ -2,11 +2,19 @@
 from django.contrib.auth.models import AbstractUser, BaseUserManager
 from django.utils.translation import gettext_lazy as _
 
-from .groups import STAFF_GROUP_NAME
+from .groups import REVIEWER_GROUP_NAME, STAFF_GROUP_NAME
 from .utils import send_activation_email
 
 
-class UserManager(BaseUserManager):
+class UserQuerySet(models.QuerySet):
+    def staff(self):
+        return self.filter(groups__name=STAFF_GROUP_NAME)
+
+    def reviewers(self):
+        return self.filter(groups__name=REVIEWER_GROUP_NAME)
```

Comment:
I am increasingly convinced this should be permission-drive. One to discuss in person